client/src/components/DataFiles/DataFilesStatus/DataFilesStatus.js

```diff
@@ -42,16 +42,31 @@ const OPERATION_MAP = {
           .slice(0, -1)
           .join('/');
         const projectName = findProjectTitle(projectList, response.systemId);
+
+        const srcSystem =
+          response.source.split('/')[0] === 'https:'
+            ? response.source.split('/')[7]
+            : response.source.split('/')[2];
```

Comment:
@duckonomy Can you tell me what these two cases are?

Reply:
I think this might not be a generic solution though...
Would there be a way to get the systemId of the source system path?

Reply:
@rstijerina The caller of `toastMap()` (In `Toast.js`) gets the response from the backend.
The response looks something like this:
```
response: {
  internalUsername: null
  lastModified: "2021-10-14T13:18:56.979-05:00"
  name: "testfile.txt"
  nativeFormat: "raw"
  owner: "ipark"
  path: "testfile.txt"
  source: "agave://cloud.corral.work.ipark/testfile.txt"
  status: ""
  systemId: "cep.local.project.ipark.CEP-4"
  uuid: ""`
}
```
I don't think it gives the source system directly.

Reply:
@rstijerina Weird, I can't seem to reproduce the https-prefixed source I was getting earlier.

Reply:
@duckonomy can you add a test to cover this case?

Reply:
@rstijerina Gotcha. I'm not sure what the https-prefixed source looked like exactly, but I'll add the test case.

Reply:
@rstijerina I've added the test cases (I've found the https-prefixed string).

I also added back the `if (mappedOp === 'copied') {` check, which I also thought was redundant.
But due to the switch-case statement falling through for copy/move/upload operations, it was necessary to check and use it only for the copy operation.